packages/js-example-app/src/index.js

```diff
@@ -5,7 +5,8 @@ const toggleLoadingButton = document.querySelector('#toggleLoadingState');
 const toggleDisabledState = document.querySelector('#toggleDisabledState');
 const logsContainer = document.querySelector('#logs');
 
-const blrButton = document.getElementsByTagName('blr-button-text')[0];
+const blrTextButton = document.getElementsByTagName('blr-text-button')[0];
+const blrIconButton = document.getElementsByTagName('blr-buton-icon')[0];
```

Comment:
name inconsistency in the tag ('blr-buton-icon') and the const variable, please check it once?

Reply:
Well spotted. That's fixed now